spec/refresher_spec.rb

```diff
@@ -5,78 +5,78 @@
   context "guess the collection" do
 
     it "Yes, it is a Object, but which class" do
-      expect([].is_a? _fill_in_object_).to be true
-      expect(Array.new.instance_of? _FILL_ME_IN_).to be true
+      expect([].is_a? Object).to be true
```

Comment:
You can be more specific than `Object` here.

Everything is an instance of `Object` in Ruby BTW. 😄